crates/biome_formatter/src/printer/mod.rs

```diff
@@ -807,29 +861,23 @@ impl Indention {
         match self {
             Indention::Level(count) => Indention::Level(count + 1),
             // Increase the indent AND convert the align to an indent
-            Indention::Align { level, .. } if indent_style.is_tab() => Indention::Level(level + 2),
+            Indention::Align {
+                level,
+                previous_align_count,
+                ..
+            } if indent_style.is_tab() => Indention::Level(level + previous_align_count + 2),
```

Comment:
```suggestion
            } if indent_style.is_tab() => Indention::Level(level + previous_align_count + 1),
```

Reply:
Thanks for the review, it’s been really helpful.

I think the variable name `previous_align_count` is confusing.
When I named the variable 'previous_align_count', I intended it to mean that it does not include the current align count.

I will show an example below.

> assuming the indent stack [Indent, Align(2), Align(2)]
> **indent**
> ```
> Indention::Level(1)
> ```
> **align(2)**
> ```
> Indention::Align{
>     level:1
>     align:2
>     previous_align_count:0
> }
> ```
> **align(2)**
> ```
> Indention::Align{
>     level:1
>     align:4
>     previous_align_count:1
> }
> ``` 

I want to choose a variable name that is not confusing.

I have some suggestion
1. How about changing the variable name from 'previous_align_count' to 'align_count', and including the current align in the count?
2. Add comment `The variable previous_align_count does not include the current align count.` and not including the current align in the count?
3. Add comment `The variable previous_align_count include the current align count.` and including the current align in the count?

If you have a better idea, let me know.

Reply:
> How about changing the variable name from 'previous_align_count' to 'align_count', and including the current align in the count?

I am in favor of this suggestion since `align_count` is not be used until it meets an `Indent`